core/tests/test_utils.py

```diff
@@ -457,6 +534,11 @@ def save_new_exp_with_states_schema_v0(self, exp_id, user_id, title):
         the usual functions for updating and creating explorations. This is
         because the latter approach would result in an exploration with the
         *current* states schema version.
+
+        Args:
+            exp_id: str. The exploration Id.
```

Comment:
Be consistent: use id or ID throughout the PR. But not both.

Reply:
Done!